apps/web/components/ui/form/LocationSelect.tsx

```diff
@@ -40,11 +42,13 @@ export default function LocationSelect(props: Props<LocationOption, false, Group
       name="location"
       id="location-select"
       components={{
-        Option: (props) => (
-          <components.Option {...props}>
-            <OptionWithIcon icon={props.data.icon} label={props.data.label} />
-          </components.Option>
-        ),
+        Option: (props) => {
+          return (
+            <components.Option {...props}>
+              <OptionWithIcon icon={props.data.icon} label={props.data.label} />
+            </components.Option>
+          );
+        },
```

Comment:
Looks like we are not supporting credentialId in here. We can revert this change.

Also @joeauyeung are we planning to enable Video Apps for teams with this PR?

Reply:
Adding video apps will be in a separate PR as it requires more designs.